### PR TITLE
homapage stat hookup

### PR DESCRIPTION
**Pull request by jaserud**
**Description of changes**
- Use stat bar query in homepage and hock up to component
- Some UI adjustments

**Type of Change**

- [ ] Bug
- [x] Styling
- [x] New Feature

**Checklist before requesting review:**

- [x] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [ ] Add copyrights to new files
- [ ] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

**Comment by hlminh2000** [comment]
if you're wondering what is going on with the test, just rebase against latest `develop` :)

**Comment by jaserud** [comment]
> if you're wondering what is going on with the test, just rebase against latest `develop` :)

thanks for the heads up :+1:

File: components/pages/Homepage/common.tsx [hunk 1]
```diff
@@ -74,7 +76,16 @@ export const DataReleaseBar: React.ComponentType<{
...
   ));
-  return (
+  return loading ? (
+    <DnaLoader
+      css={css`
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        height: 100px;
+      `}
```
Comment (hlminh2000) [inline components/pages/Homepage/common.tsx:86]: why are we applying style to this one? if we're trying to position the `DnaLoader` it makes more sense to style a container that positions the `DnaLoader` as its content eh

Comment (jaserud) [reply components/pages/Homepage/common.tsx:86]: :thinking: hmm thats a valid point, will update

File: components/pages/file-repository/StatsCard/index.tsx [hunk 2]
```diff
@@ -82,10 +91,10 @@ const StatsCard = () => {
         `}
       >
         <Col md={3} sm={6}>
```
Comment (hlminh2000) [inline components/pages/file-repository/StatsCard/index.tsx:93]: we might wanna update these columns so it flows properly with 3 items

Comment (jaserud) [reply components/pages/file-repository/StatsCard/index.tsx:93]: for homepage?

Comment (hlminh2000) [reply components/pages/file-repository/StatsCard/index.tsx:93]: sorry, yes I commented on wrong file :P

Comment (jaserud) [reply components/pages/file-repository/StatsCard/index.tsx:93]: right, the homepage was using `Row` which has a justify variable I just set that to center and that did it https://github.com/icgc-argo/platform-ui/blob/7999f4f6ac9bace779cbe5f563daeb5dc1c05232/components/pages/Homepage/common.tsx#L109

File: components/pages/Homepage/common.tsx [hunk 3]
```diff
@@ -43,9 +44,10 @@ type DataReleaseVersion = {
 export const DataReleaseBar: React.ComponentType<{
   stats: Array<DataReleaseStatistic>;
   version: DataReleaseVersion;
-}> = ({ stats, version }) => {
+  loading: boolean;
+}> = ({ stats, version, loading }) => {
   const theme = useTheme();
-  const formattedStats = stats.map(stat => (
+  const formattedStats = stats.map((stat) => (
     <Col md={3} sm={6} key={stat.description}>
```
Comment (hlminh2000) [inline components/pages/Homepage/common.tsx:51]: here's what I meant, this looks a bit weird in smaller screen:
Screen Shot 2020-06-16 at 2 54 10 PM

Comment (jaserud) [reply components/pages/Homepage/common.tsx:51]: ah now I get what you mean

**Pull request merged**